src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/TestFixturesFunctionalSpec.groovy

```diff
@@ -0,0 +1,21 @@
+package info.solidsoft.gradle.pitest.functional
+
+import groovy.transform.CompileDynamic
+import nebula.test.functional.ExecutionResult
+
+@CompileDynamic
+class TestFixturesFunctionalSpec extends AbstractPitestFunctionalSpec {
+
+    void "should work with kotlin and test fixtures"() {
```

Comment:
`kotlin and test fixtures`? That project seem to have classes written in Java and build file in Groovy :).

Reply:
Yes, it's a copy-paste mistake, I will fix it.